lib/Echidna/Mutator.hs

```diff
@@ -0,0 +1,86 @@
+module Echidna.Mutator where
+
+import Control.Monad.Random.Strict (fromList, MonadRandom, getRandomR)
+import Data.Maybe (maybe)
+import Data.ByteString (ByteString, pack, unpack)
+import GHC.Word (Word8)
+import EVM.ABI (AbiValue(..))
+
+listMutators :: MonadRandom m => m ([a] -> m [a])
+listMutators = fromList [(return, 1), (expandRandList, 10), (deleteRandList, 10), (swapRandList, 10)] 
```

Comment:
can we config these weights as well?

Reply:
I need to experiment with these parameters (with the exception of the `return` one, which should be small), in order to understand which mutations are more useful. Probably @agroce will be interested on this as well.